core/domain/stats_domain_test.py

```diff
@@ -693,60 +660,7 @@ def test_from_backend_dict(self):
         with self.assertRaisesRegexp(
             utils.ValidationError,
             'issue_type not in exploration issue dict.'):
-            stats_domain.ExplorationIssue.from_backend_dict(exp_issue_dict)
-
-    def test_update_exp_issue_from_model(self):
```

Comment:
Also, please don't remove this test. It's not part of what this PR is supposed to do, and we do want a migration test here until we actually implement a v2 for real.

Reply:
Oh. Sorry about that. Added it back.

Reply:
It doesn't look like it's been readded...

Reply:
I removed this test because it tests how the method "_convert_issue_v1_dict_to_v2_dict" works, but this method is implemented inside a test. Take a look at the removed method "_dummy_convert_issue_v1_dict_to_v2_dict" which is used in test.
Additionally, the test is using type "EarlyQuit1" which is not supported by the system right now. That's why the test became red when I added validations.
I don't see any value in this test.

Reply:
Ah, I see your point. I apologize, I didn't realize it was messing up your work in this PR.

On reflection, I guess we could add the test when we add our first conversion function, so I'm fine with what you've done here. Thanks for explaining!